pkgs/by-name/eq/equibop/package.nix

```diff
@@ -0,0 +1,154 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  substituteAll,
+  makeBinaryWrapper,
+  makeWrapper,
+  makeDesktopItem,
+  copyDesktopItems,
+  equicord,
+  electron,
+  libicns,
+  pipewire,
+  libpulseaudio,
+  autoPatchelfHook,
+  pnpm_9,
+  nodejs,
+  nix-update-script,
+  withTTS ? true,
+  withMiddleClickScroll ? false,
+  # Enables the use of Equicord from nixpkgs instead of
+  # letting Equibop manage it's own version
+  withSystemEquicord ? false,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "equibop";
+  version = "2.0.9";
+
+  src = fetchFromGitHub {
+    owner = "Equicord";
+    repo = "Equibop";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-mK/zoW8Km6xlppxJnVbuas4yE1rpAOd9QnjETlxxnsE=";
+  };
+
+  pnpmDeps = pnpm_9.fetchDeps {
+    inherit (finalAttrs)
+      pname
+      version
+      src
+      patches
+      ;
+    hash = "sha256-2YUlkclcaZNNH12Vtp6srLgxLwXd6s2QxcoB73cIrWc=";
```

Comment:
Please rebase and regenerate hashes.

See https://github.com/NixOS/nixpkgs/pull/349093#issuecomment-2421873637

Reply:
Done and done.